docs/How the library works/field-library.md

```diff
@@ -1,155 +0,0 @@
-# Field library
-
-We have many common field and widgets that are available for use in forms. You should stick with these common fields and definitions if at all possible.
-
-There are generally two forms of common definitions: schema/uiSchema objects and functions that return schema/uiSchema objects. For the function versions, there will be documentation in the fields for the parameters.
-
-## Widgets
-
-There are a set of common widgets that are included by default. You can use these by setting `ui:widget` for a field to the name. Some of them are associated with particular schema types or formats. Widgets are located in `src/js/common/schemaform/widgets`.
-
-Widget       | Default schema type or format
------------- | -------------
-TextWidget | type: string
-SelectWidget | type: string with an enum property
-RadioWidget |
-CheckboxWidget | type: boolean
-yesNo |
-
-There are more widgets than just these, but we normally use definitions they're included in, rather than just the field. These are the ones you're likely to have to set at some point in a config file.
-
-## Definitions
```

Comment:
This went to https://github.com/usds/us-forms-system/pull/112/files#diff-88f3e18582f4e60f773836345bdf7ff7R30.